web/src/main/java/nl/tehdreamteam/se42/web/command/impl/StopServicesCommand.java

```diff
@@ -0,0 +1,35 @@
+package nl.tehdreamteam.se42.web.command.impl;
+
+import nl.tehdreamteam.se42.web.ServiceContainer;
+import nl.tehdreamteam.se42.web.command.Command;
+import nl.tehdreamteam.se42.web.command.CommandHandler;
+
+public class StopServicesCommand extends Command {
+
+    private final ServiceContainer container;
+
+    public StopServicesCommand(ServiceContainer container) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Document this public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)